cmd/kube-controller-manager/app/core.go

```diff
@@ -704,6 +704,14 @@ func startGarbageCollectorController(ctx context.Context, controllerContext Cont
 	// the garbage collector.
 	go garbageCollector.Sync(ctx, discoveryClient, 30*time.Second)
 
+	// start SVM controller
+	if utilfeature.DefaultFeatureGate.Enabled(features.StorageVersionMigrator) {
```

Comment:
Why is this embedded in the start function for the garbage collector.  I expected it to stand alone.  Perhaps (I haven't checked) it needs a reference to something shared with the garbage collector (we pass those things around on the ControllerContext, peer to InformerFactory), but embedding here doesn't look right.